src/components/editor/property/InputLiteral.jsx

```diff
@@ -75,6 +75,7 @@ const InputLiteral = (props) => {
   }
 
   // TextareaAutosize does not disable well, so using a disabled input.
+
```

Comment:
```suggestion
```

d'oh, this extra line break was editing cruft on my part, feel free to squash in this suggestion for deleting the new line if its bothersome.

Reply:
If eslint doesn't complain than I'm fine with the line break :-).